spec/format_parser_spec.rb

```diff
@@ -101,4 +101,18 @@
       }.to raise_error(/No parsers provide/)
     end
   end
+
+  describe 'maximum bytes variables' do
```

Comment:
To be honest I would not check that constants are being used and I would not check their constants. We can verify behavior, and if behavior is proven to be correct it probably is all we care about. The constants we extract for readability, and for testing readability we have Rubocop already policing us :-)